v1.1/import.md

```diff
@@ -0,0 +1,366 @@
+
+Because CockroachDB is designed as a distributed system, the ergonomics of local file storage require some understanding to use successfully. Though we don't recommend this process, if you do want to use a locally stored file, this procedure is likely to cause you the fewest headaches:
+
+1. Ensure the node you want to use has available storage space at least 2x the size of the data you want to import; 1x for the file itself, and 1x for the converted key-value data.
+
+    For example, if you want to import 10GiB of data, your node needs 20GiB of available storage.
+2. Upload the tabular data file to a single node, and then connect to that node.
+3. Execute the `IMPORT` statement, importing to the locally stored file with the `nodelocal` prefix, e.g. `nodelocal://backup.csv`.
+
+    However, the "temp" directory you choose must use a location available to all nodes in the cluster (i.e., you cannot use local file storage). You will need to use either cloud storage, a custom HTTP server, or NFS connected to all nodes in the cluster.
+
+### Temp Directory
+
+To distribute the data you want to import to all nodes in your cluster, the `IMPORT` process requires the CockroachDB-compatible key-value data be stored in a location that is accessible to all nodes in the cluster using the same address. To achieve this you can use:
+
+- Cloud storage, such as Amazon S3 or Google Cloud Platform
+- Network file storage mounted to every node
+- HTTP file server
+
+{{site.data.alerts.callout_info}}If you don't currently have any of these options available, you can easily <a href="create-a-file-server.html">create a file server</a>.{{site.data.alerts.end}}
+
+The temp directory must have at least as much storage space as the size of the data you want to import.
+
+#### Temp Directory Cleanup
+
+After completing the `IMPORT` process, you must manually remove the key-value data stored in the temp directory.
+
+### Table Users and Privileges
+
+Imported tables are treated as new tables, so you must [`GRANT`](grant.html) privileges to them.
+
+## Performance
+
+Currently, `IMPORT` uses a single node to convert your tabular data into key-value data, which means the node's CPU and RAM will be partially consumed by the `IMPORT` task in addition to serving normal traffic.
+
+Later steps of the import process distribute work among many nodes and have less impact on the nodes' resources.
+
+## Synopsis
+
+{% include sql/{{ page.version.version }}/diagrams/import.html %}
+
+## Required Privileges
+
+Only the `root` user can run `IMPORT`.
+
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| **table_name** | The name of the table you want to import/create. |
+| **create_table_file** | The URL of a plain text file containing the [`CREATE TABLE`](create-table.html) statement you want to use. |
```

Comment:
Should we link to the create table from file example, as we've done for the create table from table definition?

Reply:
Done.